File: java-source/src/test/java/net/corda/accord/contract/PromissoryNoteIssueTests.java
package net.corda.accord.contract;

import net.corda.core.contracts.*;
import net.corda.testing.node.MockServices;

import net.corda.core.transactions.LedgerTransaction;
import net.corda.accord.state.PromissoryNoteState;

import java.util.Arrays;

/**
 * TODO: Modify tests to adapt from IOU to promissory note.
 */
public class PromissoryNoteIssueTests {
    // A pre-defined dummy command.
    public interface Commands extends CommandData {
        class DummyCommand extends TypeOnlyCommandData implements Commands{}
    }

    static private final MockServices ledgerServices = new MockServices(Arrays.asList("net.corda.accord", "net.corda.finance"));


}

File: kotlin-source/src/test/kotlin/net/corda/accord/contract/PromissoryNoteIssueTests.kt
package net.corda.accord.contract

import net.corda.core.contracts.TypeOnlyCommandData
import net.corda.testing.node.MockServices


class PromissoryNoteIssueTests {
    // A pre-defined dummy command.
    class DummyCommand : TypeOnlyCommandData()
    private var ledgerServices = MockServices(listOf("net.corda.accord"))


}
